src/routes/+page.svelte

```diff
@@ -349,109 +358,42 @@
             </div>
         </div>
     </div>
-
 </section>
 
 <section
     class="container flex flex-col justify-center gap-y-20 px-6 lg:px-20 py-32"
 >
-    <div class="max-w-[320px] max-w-[480px]">
+    <div class="max-w-lg">
         <h2 class="font-semibold text-5xl lg:text-7xl tracking-tight">
             A huge thanks to our supporters
         </h2>
     </div>
 
     <div
-        class="container grid gap-8 lg:grid-cols-2 md:grid-cols-3 lg:grid-cols-4"
+        class="max-screen-[1280px] mx-auto grid gap-y-6 gap-x-8 justify-items-center md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4"
     >
-        <div
-            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
-        >
-            <EigenLayer />
-
-            <div
-                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
-            >
-                <Visit />
-            </div>
-        </div>
-        <div
-            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
-        >
-            <MegaEth />
-
-            <div
-                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
-            >
-                <Visit />
-            </div>
-        </div>
-        <div
-            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
-        >
-            <img src="/Vitalik.webp" alt="" width="296" height="296" />
-
-            <div
-                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
-            >
-                <Visit />
-            </div>
-        </div>
-        <div
-            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
-        >
-            <EigenLayer />
-
-            <div
-                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
-            >
-                <Visit />
-            </div>
-        </div>
-        <div
-            class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
-        >
-            <EigenLayer />
-
+        {#each icons as icon}
             <div
-                class="absolute bottom-4 right-4 group-hover:inline-flex opacity-0 group-hover:opacity-100 transition-opacity duration-300"
+                class="relative group cursor-pointer hover:bg-gray-lightest w-[296px] h-[296px] inline-flex justify-center items-center border"
```

Comment:
We still have a little bit of overlap between 1280px to 1311px

<img width="1276" alt="image" src="https://github.com/user-attachments/assets/72769abd-5dca-46e7-93db-ed1924dd5502">